src/windows/main_window.py

```diff
@@ -565,22 +564,26 @@ def actionOpen_trigger(self):
             recommended_path,
             _("OpenShot Project (*.osp)"))[0]
 
+        s.setDefaultPath(s.actions.LOAD,file_path)
+
         # Load project file
         self.OpenProjectSignal.emit(file_path)
 
     def actionSave_trigger(self):
         app = get_app()
+        s = app.get_settings()
         _ = app._tr
 
         # Get current filepath if any, otherwise ask user
         file_path = app.project.current_filepath
         if not file_path:
-            recommended_path = os.path.join(info.HOME_PATH, "%s.osp" % _("Untitled Project"))
+            recommended_path = s.getDefaultPath(settings.actions.SAVE)
```

Comment:
As mentioned above, I would add on the project_name or "Untitled" name here, instead of inside the function.